CAN-Filter-Guide/README.adoc

```diff
@@ -0,0 +1,41 @@
+== CAN Message Filter Guide
+----
+This is a basic guide to making a CAN message filter
+----
+
+==== Main Idea
```

Comment:
Be conventional, in documentation this kind of section would be called an introduction.